.github/workflows/close-stale-needs-more-info.yml

```diff
@@ -0,0 +1,78 @@
+name: 'Close Issues with Stale Needs More Info Label'
+
+on:
+  schedule:
+    # Run this workflow at 00:00 every day
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  close-stale-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        
+      - name: Close Stale Issues with Needs More Info Label and Comment
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const labelName = 'needs-more-info';
```

Comment:
we have two needs more info labels.  Fine with using this one, but we should transition issues under the old one to it and delete the old one 
<img width="373" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/5749229/17ee1a8a-62d4-4074-982c-7cff968e215e">

Reply:
All our new labels seem to be of the space format. Do we want to keep that and delete the hyphenated version? The hyphenated has 0 issues under it anyway

Reply:
Fine with either.  easiest is probably just to use the one with spaces since we on't have to move labels